modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -191,6 +191,13 @@ If you do so, you must add ingress DNS records manually in a later step.
 endif::user-infra-vpc[]
 endif::gcp,aws,azure[]
 
+ifdef::vsphere[]
+. Provide the name of the vCenter folder that hosts your cluster's virtual machines to the cloud provider:
+.. Open the `<installation_directory>/manifests/cloud-provider-config.yaml` file.
+.. Set the value of the `folder` parameter to match the value that you specified for the folder name in the `install-config.yaml` file or, if you did not specify a folder name, the cluster ID.
```

Comment:
This section is not quite right, and I don't think we actually need it. This line is describing what the `folder` value in the cloud-provider-config will already be--no users steps are needed to achieve this.